src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/Baselines/TestConfigureCallGen.generated.txt

```diff
@@ -0,0 +1,91 @@
+// <auto-generated/>
+#nullable enable
+
+using System.Linq;
+
+internal static class GeneratedConfigurationBinder
+{
+    public static global::Microsoft.Extensions.DependencyInjection.IServiceCollection Configure<T>(this global::Microsoft.Extensions.DependencyInjection.IServiceCollection services, global::Microsoft.Extensions.Configuration.IConfiguration configuration)
+    {
+        if (typeof(T) == typeof(Program.MyClass))
+        {
+            return services.Configure<Program.MyClass>(obj =>
+            {
+                BindCore(configuration, ref obj);
+            });
+        }
+
+        throw new global::System.NotSupportedException($"Unable to bind to type '{typeof(T)}': 'Generator parser did not detect the type as input'");
+    }
+
+    private static void BindCore(global::Microsoft.Extensions.Configuration.IConfiguration configuration, ref Program.MyClass obj)
+    {
+        if (obj is null)
+        {
+            throw new global::System.ArgumentNullException(nameof(obj));
+        }
+
+        if (configuration["MyString"] is string stringValue1)
```

Comment:
optional feedback:
one suggestion, for variable names I suggest to create subroutine like:

`GenerateSymbol(string someLabel)` and basically just use `someLabel` with suffix, here probably `myString` or `MyString` or `MyString` or something similar might be better variable name